compiler/noirc_frontend/src/elaborator/trait_impls.rs

```diff
@@ -59,7 +62,9 @@ impl<'context> Elaborator<'context> {
 
                     let func_id = self.interner.push_empty_fn();
                     let module = self.module_id();
-                    let location = Location::new(default_impl.def.span, trait_impl.file_id);
+                    // TODO: check this, it doesn't make sense to mix locations from two different files
```

Comment:
Please make a follow-up issue for this TODO

Reply:
Done: https://github.com/noir-lang/noir/issues/7454